### PR TITLE
Add Python packaging infrastructure to properly distribute jose

### PR DESCRIPTION
**Pull request by tiran**
Signed-off-by: Christian Heimes cheimes@redhat.com

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.. WARNING: AUTO-GENERATED FILE. DO NOT EDIT.
```
Comment (npmccallum) [inline README:1]: I don't see anything that generates this file.

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Python bindings for libjose
```
Comment (npmccallum) [inline README.md]: Can we give this file better content? Preferably including a link to jose.

File: README (hunk 1 above)
Comment (tiran) [reply README:1]: I'll add the Makefile with the build rule in another PR.

File: README.md (hunk 2 above)
Comment (tiran) [reply README.md]: PR welcome :)

**Pull request merged**